aerogear/ionic-showcase
# feat: tools to simplify local keycloak setup

### Description



This PR makes working locally with keycloak + the showcase app really easy.

* adds a `npm run keycloak` command that starts a keycloak server (using docker-compose)
* adds a `npm run keycloak:init` command that imports a realm, creates roles + users and generates the config needed for the server (`keycloak.json`) and the service config (`mobile-services.json`) for the mobile application.

## Verification Steps

There are steps for setting up the keycloak integration in the readme.

* Follow the steps in the readme to 
  * spin up keycloak
  * run the `keycloak:init` script
  * copy the appropriate configs into the correct location.
* Start the server and the app
* You should be brought to the keycloak login screen
* You can log in with either `developer/developer` or `admin/admin`

##### Checklist
